javascript/index.js

```diff
@@ -56,3 +89,30 @@ class BookStore {
 
 // eslint-disable-next-line no-unused-vars
 const bookStore = new BookStore();
+
+// const showContact = document.getElementById('contact');
+// const showAddBook = document.querySelector('#form');
+// const showBookList = document.querySelector('');
+// console.log(showContact);
+
+// navAddBook.addEventListener('click', () => {
+//   showContact.style.display = 'none';
+//   showAddBook.style.display = 'flex';
+//   showBookList.style.display = 'none';
+// console.log('add book');
+// });
+
+// navbookList.addEventListener('click', () => {
+//   showContact.style.display = 'none';
+//   showAddBook.style.display = 'none';
+//   showBookList.style.display = 'flex';
+// console.log('show books');
+// });
+
+// navContact.addEventListener('click', () => {
+//   showContact.style.display = 'flex';
+//   showContact.style.display = 'none';
+//   showAddBook.style.display = 'none';
+//   showBookList.style.display = 'flex';
+// console.log('show contact');
+// });
```

Comment:
- [ ] please always do well to remove commented code from your code base so as to not make your code messy and unprofessional 😉

Reply:
This change has been done well